core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Ellipse2afp.java

```diff
@@ -56,6 +56,11 @@
 		B extends Rectangle2afp<?, ?, IE, P, V, B>>
 		extends RectangularShape2afp<ST, IT, IE, P, V, B> {
 
+    final String ELLIPSE_WIDTH_POSITIVE_OR_ZERO = "Ellipse width must be positive or zero";
```

Comment:
I agree with you. Please clean this PR in order to be merged asap. We will introduces the `AssertionMessages` class at a second stage, as stated by the new issue #56.